Home_work_11/home_work_11.py

```diff
@@ -0,0 +1,35 @@
+# Даны две функции:
+# f(x) = x^3 - 50x и g(x) = -x^4 + 88x^2 - 241
+# Требуется:
+# 1. Найти координаты точек пересечения
+# 2. Построить графики функций в одной системе координат
+# 3. Построить график функции пересечения
+
+from sympy import *
+from sympy.plotting import plot
+
+init_printing()
+
+x = Symbol('x', real=True)
+
+# уравнения
+f = x**3 - 50*x
+g = (-x)**4 + (88*x)**2 - 241
```

Comment:
Вот тут неверно, из за этого у вас неверно посчитано. У этих графиков 4 точки пересечения.
```suggestion
f = x ** 3 - 50 * x
g = -x ** 4 + 88 * x ** 2 - 241
```